pycbc/inference/sampler/nessai.py

```diff
@@ -98,6 +94,15 @@ def run(self, **kwargs):
         extra_kwds = self.extra_kwds.copy()
         run_kwds = self.run_kwds.copy()
 
+        # Output in kwargs takes priority of extra kwds.
+        output = kwargs.pop("output", extra_kwds.pop("output", None))
+        # If neither have been specified, use the path from the checkpoint file
+        if output is None:
```

Comment:
@mj-will So if you don't set an output dir, it will still create one? Is that intentional?

Reply:
@ahnitz yes, nessai takes an output directory as an input rather than a file, so I opted to have it create one by default rather than dumping everything into the current working directory. I don't have a strong preference though and happy to change it.

Reply:
Ah, ok, so that's really no different than the current situation.